language/analysis/src/main/java/com/google/cloud/language/samples/AnalyzeBeta.java

```diff
@@ -67,109 +53,171 @@ public static void main(String[] args) throws IOException, GeneralSecurityExcept
    */
-  public Sentiment analyzeSentimentText(String text, String lang) throws IOException {
-    // NL autodetects the language
-    Document doc;
-    if (lang != null) {
-      doc = Document.newBuilder()
-          .setLanguage(lang)
-          .setContent(text).setType(Type.PLAIN_TEXT)
+  public static void entitySentimentFile(String gcsUri) throws Exception {
+    // [START entity_sentiment_file]
+    // Instantiate a beta client : com.google.cloud.language.v1beta2.LanguageServiceClient
+    try (LanguageServiceClient language = LanguageServiceClient.create()) {
+      Document doc = Document.newBuilder()
+          .setGcsContentUri(gcsUri)
+          .setType(Type.PLAIN_TEXT)
           .build();
-    } else {
-      doc = Document.newBuilder()
-          .setContent(text).setType(Type.PLAIN_TEXT)
+      AnalyzeEntitySentimentRequest request = AnalyzeEntitySentimentRequest.newBuilder()
+          .setDocument(doc)
+          .setEncodingType(EncodingType.UTF16)
           .build();
+      // Detect entity sentiments in the given file
+      AnalyzeEntitySentimentResponse response = language.analyzeEntitySentiment(request);
+      // Print the response
+      for (Entity entity : response.getEntitiesList()) {
+        System.out.printf("Entity: %s\n", entity.getName());
+        System.out.printf("Salience: %.3f\n", entity.getSalience());
+        System.out.printf("Sentiment : %s\n", entity.getSentiment());
+        for (EntityMention mention : entity.getMentionsList()) {
+          System.out.printf("Begin offset: %d\n", mention.getText().getBeginOffset());
+          System.out.printf("Content: %s\n", mention.getText().getContent());
+          System.out.printf("Magnitude: %.3f\n", mention.getSentiment().getMagnitude());
+          System.out.printf("Sentiment score : %.3f\n", mention.getSentiment().getScore());
+          System.out.printf("Type: %s\n\n", mention.getType());
+        }
+      }
     }
-    AnalyzeSentimentResponse response = languageApi.analyzeSentiment(doc);
-    return response.getDocumentSentiment();
+    // [END entity_sentiment_file]
   }
 
   /**
-   * Gets {@link Entity}s from the string {@code text} with sentiment.
+   * Detects categories in text using the Language Beta API.
    */
-  public List<Entity> entitySentimentText(String text) throws IOException {
-    Document doc = Document.newBuilder()
-            .setContent(text).setType(Type.PLAIN_TEXT).build();
-    AnalyzeEntitySentimentRequest request = AnalyzeEntitySentimentRequest.newBuilder()
-            .setDocument(doc)
-            .setEncodingType(EncodingType.UTF16).build();
-    AnalyzeEntitySentimentResponse response = languageApi.analyzeEntitySentiment(request);
-    return response.getEntitiesList();
+  public static void classifyText(String text) throws Exception {
+    // [START classify_text]
+    // Instantiate a beta client : com.google.cloud.language.v1beta2.LanguageServiceClient
```

Comment:
Do we need this as long as this?

Reply:
Pre-dates this release, but looks like we are maintaining the 
`com.google.cloud.language.v1beta2.LanguageServiceClient`
and `com.google.cloud.language.v1.LanguageServiceClient` in parallel with new features being addded in v1beta2. 
I provided the import path here, so it is clear to the user which one we are using in these snippets.